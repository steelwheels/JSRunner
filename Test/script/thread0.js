/*
 * thread0.js
 */

console.log("start\n") ;
let t0 = thread(function() {
	console.log("Message from thread\n") ;
	return 0 ;
}) ;
//console.log("thread: " + t0 + "\n") ;
t0.waitUntilExit() ;
console.log("bye\n") ;

