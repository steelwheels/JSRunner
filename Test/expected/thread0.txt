Hello from main function
start: thread0
start: wait until exit
Hello from thread0
args: a,b
exit code = 0
